opensrp-reveal/src/test/java/org/smartregister/reveal/shadow/CloudantDataHandlerShadow.java

```diff
@@ -0,0 +1,21 @@
+package org.smartregister.reveal.shadow;
+
+import android.content.Context;
+
+import org.mockito.Mockito;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+import org.smartregister.sync.CloudantDataHandler;
+
+/**
+ * Created by Richard Kareko on 5/29/20.
+ */
+
+@Implements(CloudantDataHandler.class)
+public class CloudantDataHandlerShadow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'CloudantDataHandlerShadow' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5569929)